casino/repository/blockchain/casino.go

```diff
@@ -19,7 +19,7 @@ type Casino struct {
 
 type GameParam struct {
 	Type  uint16 `json:"first"`
-	Value uint32 `json:"second"`
+	Value uint64 `json:"second"`
```

Comment:
for now we use uint32_t as param value, to uint64_t we will move in further release